rs_workflows/staging.py

```diff
@@ -92,7 +93,12 @@ def update_stac_catalog(  # pylint: disable=too-many-locals
     now = datetime.now()
 
     # The file path from the temp s3 bucket is given in the assets
-    assets = {"file": Asset(href=f"{obs.rstrip('/')}/{stac_file_info['id']}")}
+    if kwargs.get("update_assets", None):
```

Comment:
And is it really necessary to pass update_assets as a parameter ? Can't we just do this ?
```python
update_assets = len(stac_file_info["assets"]) > 1
```

Reply:
You need to merge assets only if flag **update_assets** is set.